### PR TITLE
Update GET_YOUR_SESSION_ID.md

### PR DESCRIPTION
**Pull request by JackAttack365**
Adds more detail and browser specific content, and updates the alternate method

### CODE REVIEW

**Review by mas6y6 — APPROVED** [review]
This is valid

**Review by TheCommCraft — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,6 @@ __pycache__/
 README.md
 setup.py
 scratchattach/test.py
-scratchattach.code-workspace
\ No newline at end of file
+scratchattach.code-workspace
+.DS_Store
+wiki/.DS_Store
```
Comment (namtacs) [inline .gitignore:10]: wiki/.DS_Store is not needed.
_I'm starting to think every contributor to this project doesn't know basic repository management_

**Review by namtacs — CHANGES_REQUESTED** [review]

**Review by namtacs — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (TheCommCraft) [reply .gitignore]: actually, I am not sure that's true, but you can just replace both with "**/.DS_STORE"

**Comment by SpyC0der77** [comment]
Can someone merge this?

**Comment by TimMcCool** [comment]
.

**Pull request merged**